config/environments/production.rb

```diff
@@ -83,5 +83,5 @@
 
   # Do not dump schema after migrations.
   # config.active_record.dump_schema_after_migration = false
-  config.mvi_service = MVI::Service
+  config.mvi_service = MVI::MockService
```

Comment:
This is a nitpick, but would this maybe be better as an ENV var, that way if we forget to change it for actual production vs dev-api / staging-api it would be OK. 

Totally a nitpick though as I'm sure we'll catch it during QA